pkgs/games/openmw/default.nix

```diff
@@ -1,34 +1,23 @@
-{ stdenv, fetchFromGitHub, qt4, ogre, mygui, bullet, ffmpeg, boost, cmake, SDL2, unshield, openal, pkgconfig }:
+{ stdenv, fetchFromGitHub, qt4, openscenegraph, mygui, bullet, ffmpeg, boost, cmake, SDL2, unshield, openal, pkgconfig, libX11, unzip, ois, freetype, libuuid }:
 
 stdenv.mkDerivation rec {
-  version = "0.36.1";
+  version = "0.38.0";
   name = "openmw-${version}";
 
-  mygui_ = mygui.overrideDerivation (oldAttrs: {
-    name = "mygui-3.2.1";
-    version = "3.2.1";
-
-    src = fetchFromGitHub {
-      owner = "MyGUI";
-      repo = "mygui";
-      rev = "MyGUI3.2.1";
-      sha256 = "1ic4xwyh2akfpqirrbyvicc56yy2r268rzgcgx16yqb4nrldy2p0";
-    };
-  });
+  mygui_ = mygui.override { withOgre = false; } ;
```

Comment:
Since we now build `mygui` without Ogre by default you can skip overriding this. Remark for the future: we usually override packages in `all-packages.nix` instead, for example:
```
  foo = callPackage ../bar/baz/foo {
    xyz = xyz.override {
      withAbc = false;
    };
  };
```